### DISCUSSION TITLE
Welcome to blender-sculpting-layers Discussions!

### DISCUSSION THREAD

**Discussion by FadiMHussein** [FadiMHussein/blender-sculpting-layers#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MrAdrianPl** [comment]
Hi im wondering is addon still developed as it wasn't updated for quiet some time it looks as great tool even though its rather taxing on performance with higher mesh resolutions.

**Reply by FadiMHussein** [reply]
Thank you for your interest in this addon. Yes, I am currently working on a new version for the LTS branch of blender. As for the performance issues I recommend using muti-resollution for high density mesh for the time being (Still looking for a fast way to manipulate and save mesh vertices or creating a mesh typology system from scratch). If you are interested in the development of the addon please contribute to the code in any way possible; even by sharing your thoughts on where the development should go.

**Reply by MrAdrianPl** [reply]
Im sculpting rather rarely as i preferer the "standard" modeling but layers are very useful. Im very interested in further development but i have no clue about blender api and i would have to learn it from scratch, though i know basics of python despite that I don't have much time right now to learn the api.
This may sound weird but maybe you could take a look on the blender sculpt layers addon i mean the other one im not sure how they solve it but im pretty sure they do it different way than the addon of yours. 
Also coming with more distinct name from the other one would be good idea in my opinion. The addon is hidden behind the other one on the google search 
few small suggestions for features 
-auto record checkbox would be cool as i tend to forget about clicking record button every time i create new layer
-it would be cool if 2nd mesh could be turned off in visibility
-block possibility to undo shape keys deformations after applying it, as it cause blender to crash
Also this is more of a bug report but if there's more than one sculpting layer at once you cant access previous ones as it cause working on the last one and prompts error on recording end.
Im not sure if im doing something wrong but the weights seams to not work properly, but this may be only on my end as im using shape key addon that isn't working well with other addons.

**Reply by FadiMHussein** [reply]
Your feedback is very helpful thanks a lot. I will further analyze each point and start opening issues in order to solve them and implement the features proposed. As for the other addon they are using their own system of tracking each vertex position and location. The problem with tracking is that the vertices are restructured each time you play with the mesh. However, I am still looking for ways to implement my own system for free (The other addon does cost some money). Finally for the name change I would love the community's help in naming the addon (Not that creative when it comes to naming things :) )

**Reply by MrAdrianPl** [reply]
So i have few propositions these aren't particular creative i guess but some might fit your taste
FHLayers, just first letters from your name and surname as im assuming you're using your own in the nickname
BSL[Blender Sculpting Layers], acronyms are popular and this might sound more distinctive 
Artisan Sculptor, bit less descriptive name more distinctive 
Layer Sculptor, bit more distinctive but still might be confusing
Layered, even simpler one and its easy to remember
